azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -54,7 +55,7 @@ public void run() throws Exception {
       handleError("Bad property definition! " + e.getMessage(), e);
     }
 
-    if (sysProps.getBoolean(MEMCHECK_ENABLED, true)) {
+    if (sysProps.getBoolean(MEMCHECK_ENABLED, true) && jobProps.getBoolean(AZKABAN_MEMORY_CHECK, false)) {
```

Comment:
jobProps.getBoolean(AZKABAN_MEMORY_CHECK, false)
should this be defaulted to true? so that we will by default check for memory?